modules/nested-elements/assets/js/frontend/handlers/base-nested.js

```diff
@@ -0,0 +1,70 @@
+import Base from 'elementor-frontend/handlers/base';
+
+export default class BaseNested extends Base {
```

Comment:
Move contents of this class to BaseNestedTabs and delete the class and its imports and usage

Reply:
Done